doc/api/cli.md

```diff
@@ -241,6 +241,20 @@ This flag exists to aid in experimentation with the internal implementation of
 the Node.js http parser.
 This flag is likely to become a no-op and removed at some point in the future.
 
+### `--http-server-default-timeout=milliseconds`
+<!-- YAML
+added: REPLACEME
+-->
+
+Overrides the default value of `http`, `https` and `http2` server socket
+timeout. Setting the value to 0 disables server socket timeout. Unless
+provided, http server sockets timeout after 120s (2 minutes). Programmatic
+setting of the timeout takes precedence over the value set through this
+flag.
+
+This flag is available on the recent versions of Node.js 8.x, 10.x and 12.x
+release branches only.
```

Comment:
I suggest to remove the whole comment. The information when the flag is added is already available due to the `added: ...` part.

Reply:
The reason the needs to be there is to indicate that node 13 and onwards do not have this flag.

Reply:
We have other flags that are removed in following versions without any further special documentation. Some of them might be experimental but I am relatively certain that we also have others that would not be available later on and just removed. In Node.js 13 it would not be documented which seems correct.

I guess the reason you would like to have some documentation about the specific version is to indicate that users will have to switch in future versions? I just wonder if telling the person who adds the flag will help switching to newer versions, since it'll fail either way and they probably have to read about it again during the update.

Reply:
👍 . Originally this was supposed to be an environment variable which have silently stopped working. But given that this is a command line flag, node fail give an error if used against an unsupported release. I've removed the comment as you suggested.